Entity/TaxonCategoryReference.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SyliusConsumerBundle\Entity;
+
+use Sulu\Bundle\CategoryBundle\Entity\CategoryInterface;
+
+class TaxonCategoryReference implements TaxonCategoryReferenceInterface
```

Comment:
we called these entities `Bridge` in other projects. so this would be a `TaxonCategoryBridge`